exchange/adapter_builders.go

```diff
@@ -263,7 +263,7 @@ func newAdapterBuilders() map[openrtb_ext.BidderName]adapters.Builder {
 		openrtb_ext.BidderBidmyadz:          bidmyadz.Builder,
 		openrtb_ext.BidderBidsCube:          bidscube.Builder,
 		openrtb_ext.BidderBidstack:          bidstack.Builder,
-		openrtb_ext.BidderBizzclick:         bizzclick.Builder,
+		openrtb_ext.BidderBlasto:            blasto.Builder,
```

Comment:
How about keeping Bizzclick as an alias of Blasto to allow existing configurations continue to work? Do you really want all of your existing traffic to stop working until publishers take steps to change the name?

Reply:
I don't think we need this, we'll let current publishers know about this update so it's ok